data/en/serializejson.json

```diff
@@ -1,13 +1,14 @@
 {
 	"name":"serializeJSON",
 	"type":"function",
-	"syntax":"serializeJSON(var [, serializeQueryByColumns, useCustomSerializer])",
+	"syntax":"SerializeJSON(var[, serializeQueryByColumns[, useSecureJSONPrefix[, useCustomSerializer]]])",
```

Comment:
The syntax was changed to uppercase `SerializeJSON` it should start with a lower case `serializeJSON`

Otherwise the PR looks good - sorry for taking so long to take a look, been pretty busy over the last several weeks

Reply:
No worries. We all know busy times. 
I've updated the pullrequest.